File: PersonnelManagement/PersonnelManagement/Front/classes/Excerts/excertComposition.ts
import Persondecreeoperation from '../../classes/persondecreeoperation'
import Persondecreeblock from '../../classes/persondecreeblock'
import Persondecreeblocktype from '../../classes/persondecreeblocktype'
import Persondecreeblocksub from '../../classes/persondecreeblocksub'
import Persondecreeblocksubtype from '../../classes/persondecreeblocksubtype'
import User from '../../classes/user';
import Persondecree from '../../classes/persondecree';

export default class ExcertComposition {
    decree: Persondecree;
    decreeblocks: Persondecreeblock[];
    decreeblocksubs: Persondecreeblocksub[];
    decreeblocksubtypes: Persondecreeblocksubtype[];
    decreeblocktypes: Persondecreeblocktype[];
    decreeoperations: Persondecreeoperation[];
}

File: PersonnelManagement/PersonnelManagement/Models/Excerts/ExcertComposition.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PersonnelManagement.Models
{
    public class ExcertComposition
    {
        public PersondecreeManagement decree { get; set; }
        public List<PersondecreeblockManagement> decreeblocks { get; set; }
        public List<Persondecreeblocktype> decreeblocktypes { get; set; }
        public List<Persondecreeblocksub> decreeblocksubs { get; set; }
        public List<Persondecreeblocksubtype> decreeblocksubtypes { get; set; }
        public List<PersondecreeoperationManagement> decreeoperations { get; set; }

        public ExcertComposition()
        {
            decree = new PersondecreeManagement();
            decreeblocks = new List<PersondecreeblockManagement>();
            decreeblocktypes = new List<Persondecreeblocktype>();
            decreeblocksubs = new List<Persondecreeblocksub>();
            decreeblocksubtypes = new List<Persondecreeblocksubtype>();
            decreeoperations = new List<PersondecreeoperationManagement>();
        }
    }
}
